vignettes/tidy.Rmd

```diff
@@ -13,6 +13,11 @@ knitr::opts_chunk$set(
   comment = "#>",
   rows.print = 25
 )
+
+# one of the functions below needs emmeans, so don't build vignette on old R versions where emmeans is not available
+if (!rlang::is_installed("emmeans")) {
+  knitr::knit_exit()
```

Comment:
I think `knitr::opts_chunk$set(eval = FALSE)` would be better since then you'll at least get the vignette, without the code being run.

Reply:
that is a much better alternative, thank you!